### PR TITLE
-> v1.9.1-RC1

### PR DESCRIPTION
**Pull request by mhowlett**
Notes:
- we will merge the ACL PR before releasing v1.9.1.
- .NET 6.0 is required for Apple M1 support. (.NET 5.0 is also out of support).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+
+## Enhancements
+
+- References librdkafka.redist 1.9.1 which includes an Apple M1 librdkafka build.
+- Enhanced the Avro Specific Deserializer to ignore the type namespace ([sergemat](https://github.com/sergemat)).
+
+
+## Fixes
+
+- The AdminClient poll loop no longer terminates when a request results in an error ([emasab](https://github.com/emasab)).
```
Comment (emasab) [inline CHANGELOG.md:11]: No problem if you don't reference me, as we are part of the team. You could put an hashtag with the PR for the team members, and for the rest the username too.

**Review by emasab — APPROVED** [review]

**Comment by niemyjski** [comment]
@mhowlett pr looks good, I think the builds are failing because of VS target (2019 vs 2022) in appveyor and dotnet version is 2.1 on travis?

File: CHANGELOG.md (hunk 1 above)
Comment (mhowlett) [reply CHANGELOG.md:11]: yeah, either way. when in doubt, give credit :-)

**Comment by mhowlett** [comment]
ahh, now i'm remembering the specific reason we didn't upgrade to .net 6.0 earlier - the unit tests were hanging ...

**Comment by niemyjski** [comment]
Do you know why? This could also be my problem as my foundatio tests run under .net 6

**Comment by niemyjski** [comment]
Also this seems like a great reason to upgrade to .net 6 when you first ran into this? As .net 6 is LTS...

**Comment by mhowlett** [comment]
concluded it was related to the test framework, and deferred to later ... which is now now.

**Comment by niemyjski** [comment]
I think Travis still needs to be updated to .NET 6

```
 dist: focal # Ubuntu 20.04
    dotnet: 5.0.408
```

**Comment by mhowlett** [comment]
we are in the process of discontinuing use of travis and moving to another CI system.

**Pull request merged**